Documentation/contributing/development/dev_setup.rst

```diff
@@ -152,6 +152,18 @@ code, in an pre-existing running Cilium container.
 * ``make kind-image-fast``: Builds all Cilium binaries and loads them into all
   kind clusters available in the host.
 
+Configuration for Cilium
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Makefile targets that install Cilium are passing the following list of Helm
+values (YAML files) to the Cilium CLI.
+
+* ``contrib/testing/kind-common.yaml``: Shared between normal and fast installation modes.
+* ``contrib/testing/kind-values.yaml``: Used by normal installation mode.
+* ``contrib/testing/kind-fast.yaml``: Used by fast installation mode.
+* ``contrib/testing/kind-custom.yaml``: User defined custom values that are applied if
+  the file is present. The file is ignored by Git.
```

Comment:
I like that ignoring the file is default behavior. ✨ 

**Optional:** given that `contrib/testing/.gitignore` is a new addition, it may be helpful to signal the mechanism clearly to devs with another `.gitignore`.
```suggestion
  the file is present. The file is ignored by Git as specified in ``/contrib/testing/.gitignore``.
```

Reply:
Thanks, definitely makes sense.